test/test_mysql.py

```diff
@@ -57,6 +59,22 @@ def test_lost_connection(self):
         self.assertRaises(apel.db.apeldb.ApelDbException,
                           self.db.test_connection)
 
+    def test_schema_version(self):
+        """
+        Verify that the schema version checks function correctly
+        """
+        # The database server.sql script initialises with the current versions
+        # so the initial state should be consistent
+        self.assertTrue(self.db.check_versions() is None)
```

Comment:
does `check_versions()` actually return anything? ever? Is this check actually checking that no exception is raised?